src/scss/components/dropdown-select/_mixins.scss

```diff
@@ -1,12 +1,39 @@
-@import "mixins/microsoft";
-@import "mixins/mozilla";
-@import "mixins/standard";
-@import "mixins/webkit";
-
-@mixin reset-select { // less: #Reset.select()
-  @include ms-reset-select;
-  @include moz-reset-select;
-  @include webkit-reset-select;
-  @include std-reset-select;
```

Comment:
Refactored to remove all `webkit-*`, `ms-*`, `moz-*`, and `std-*` mixins in favor of their raw CSS.

This is to simplify mixins for the purposes of migrating to SCSS.  We can re-add needed mixins later, once we've fully converted to SCSS.